docs/sources/tempo/configuration/_index.md

```diff
@@ -1506,7 +1644,35 @@ include:
   match_type: 'strict'
   attributes:
     - key: 'foo.bar'
-      value: 'baz'
+      value: "baz"
+```
+
+### GRPC client config
+
```

Comment:
Consider adding an introductory sentence here. Usually, we try to have an intro sentence between a heading and an example.

Reply:
I've pushed a commit for this.